mozdownload/scraper.py

```diff
@@ -36,7 +36,10 @@
 APPLICATIONS = ('b2g', 'firefox', 'thunderbird')
 
 # Base URL for the path to all builds
-BASE_URL = 'https://ftp.mozilla.org/pub/mozilla.org'
+BASE_URL = 'http://download.cdn.mozilla.net/pub'
+
+# In case CDN fails
+BACKUP_URL = 'https://ftp.mozilla.org/pub/mozilla.org'
```

Comment:
I don't think we have to carry the backup URL with us. @nthomas-mozilla what do you say? Is it enough to explicitly work with CDN only? I don't think there will be cases when it will not work.